ui/app/mainui/Handlers/SendModalHandler.qml

```diff
@@ -660,7 +660,8 @@ QtObject {
                     if(uuid !== handler.uuid) {
                         return
                     }
-                    simpleSendModal.close()
+                    // TODO: commented this out as closing this popup rests the variables on nim side and tx fails
+                    // simpleSendModal.close()
```

Comment:
TODO still valid?

Reply:
yes it is, we need to go through this sign flow again, because they dont seem perfect.